remote_config/src/include/firebase/remote_config.h

```diff
@@ -258,7 +258,7 @@ class RemoteConfig {
   ///
   /// @note This does not actually apply the data or make it accessible,
   /// it merely retrieves it and caches it.  To accept and access the newly
-  /// retrieved values, you must call @ref ActivateFetched().
+  /// retrieved values, you must call @ref Activate().
```

Comment:
This should have been fixed in #481. Can you do a pull to make sure your branch is up to date?

Reply:
Nevermind, looks like it auto-merged cleanly and closed the branch.